components/footer.tsx

```diff
@@ -1,35 +1,133 @@
 export default function Footer() {
+    const navigation = {
+        solutions: [
```

Comment:
I don't see where we use any of these except the "social" object key. We can remove the others since they're not useful